src/EmergencyDevice.ts

```diff
@@ -0,0 +1,71 @@
+import { t } from 'ttag';
+import SimpleSchema from 'simpl-schema';
+import './SimpleSchemaExtensions';
+import { Length, LengthSchema, quantityDefinition } from './Units';
+import { LocalizedStringSchema, LocalizedString } from './LocalizedString';
+
+/**
+ * Describes a EmergencyDevice unit provided at this place, for example an audibleFireAlarm
+ */
+export interface EmergencyDevice {
+  /**
+   * Type of the EmergencyDevice unit
+   */
+  type: 'evacChair' | 'alarm' | 'visualFireAlarm' | 'audibleFireAlarm';
+
+  /**
+   * Is the emergency device in audio
+   */
+  isAudio?: boolean;
+
+  /**
+   * Is the emergency device visual
+   */
+  isVisual?: boolean;
+
+  /**
+   * Is the emergency device visual
+   */
+  isAvailable?: boolean;
```

Comment:
I think we can remove this attribute. The availability of the device can be encoded in its existence in the model.

Reply:
Yeah I had my doubts about this one too.

Reply:
removed